preprod.healdata.org/manifests/hatchery/hatchery.json

```diff
@@ -2,209 +2,67 @@
   "user-namespace": "jupyter-pods-healpreprod",
   "sub-dir": "/lw-workspace",
   "user-volume-size": "10Gi",
-  "pay-models": [{
-      "user_id": "aprokh@uchicago.edu",
-      "aws_account_id": "252835817747",
-      "name": "healworkspaces",
-      "region": "us-east-1"
-    },
-    {
-      "user_id": "atharvar@uchicago.edu",
-      "aws_account_id": "252835817747",
-      "name": "healworkspaces",
-      "region": "us-east-1"
-    },
-    {
-      "user_id": "avantol@uchicago.edu",
-      "aws_account_id": "252835817747",
-      "name": "healworkspaces",
-      "region": "us-east-1"
-    },
-    {
-      "user_id": "emalinowski@uchicago.edu",
-      "aws_account_id": "252835817747",
-      "name": "healworkspaces",
-      "region": "us-east-1"
-    },
-    {
-      "user_id": "fan1@uchicago.edu",
-      "aws_account_id": "252835817747",
-      "name": "healworkspaces",
-      "region": "us-east-1"
-    },
-    {
-      "user_id": "haraprasadj@uchicago.edu",
-      "aws_account_id": "252835817747",
-      "name": "healworkspaces",
-      "region": "us-east-1"
-    },
-    {
-      "user_id": "mcannalte@uchicago.edu",
-      "aws_account_id": "252835817747",
-      "name": "healworkspaces",
-      "region": "us-east-1"
-    },
-    {
-      "user_id": "mpingram@uchicago.edu",
-      "aws_account_id": "252835817747",
-      "name": "healworkspaces",
-      "region": "us-east-1"
-    },
-    {
-      "user_id": "nmetokishlubsky@uchicago.edu",
-      "aws_account_id": "252835817747",
-      "name": "healworkspaces",
-      "region": "us-east-1"
-    },
-    {
-      "user_id": "ribeyre@uchicago.edu",
-      "aws_account_id": "252835817747",
-      "name": "healworkspaces",
-      "region": "us-east-1"
-    },
-    {
-      "user_id": "svburke@uchicago.edu",
-      "aws_account_id": "252835817747",
-      "name": "healworkspaces",
-      "region": "us-east-1"
-    },
-    {
-      "user_id": "mshao1@uchicago.edu",
-      "aws_account_id": "252835817747",
-      "name": "healworkspaces",
-      "region": "us-east-1"
-    },
-    {
-      "user_id": "craigrbarnes@uchicago.edu",
-      "aws_account_id": "252835817747",
-      "name": "healworkspaces",
-      "region": "us-east-1"
-    },
+  "default-pay-model": {
+    "aws_account_id": "252835817747",
+    "name": "healworkspaces",
+    "region": "us-east-1"
+  },
+  "pay-models": [
     {
       "user_id": "cgmeyer@uchicago.edu",
-      "aws_account_id": "252835817747",
-      "name": "healworkspaces",
-      "region": "us-east-1"
+      "aws_account_id": "134208290168",
+      "name": "strides_grant",
+      "region": "us-east-1",
+      "ecs": "true",
+      "subnet": 0
     },
     {
       "user_id": "xritter@uchicago.edu",
-      "aws_account_id": "252835817747",
-      "name": "healworkspaces",
-      "region": "us-east-1"
-    },
-    {
-      "user_id": "planxdemo@gmail.com",
```

Comment:
test email should stay as well

Reply:
The default paymodel will take care of everyone that are not specified under paymodels.